src/components/pages/messages/Main.js

```diff
@@ -398,6 +402,9 @@ export default function Main() {
 
   const handleSearchAccounts = text => {
     setSearch(text)
+    if (!text || text.length == 0) {
```

Comment:
setSearch() should have triggered doFetchAccounts via useEffect Line 72. Just curious.

Reply:
Line 72 is a function not a useEffect hook.